telco_ref_design_specs/ran/telco-ran-ref-validation-artifacts.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-ran-ref-validation-artifacts"]
+= Telco RAN DU reference configuration validation artifacts
+include::_attributes/common-attributes.adoc[]
+:context: ran-ref-design-validation
+
+toc::[]
+
+The following information describes the telco RAN DU reference configuration, hardware, KPI, scale, and system test validation metrics.
+
+include::modules/telco-ran-hardware-platform-specifications.adoc[leveloffset=+1]
+
+include::modules/telco-ran-ref-config-kpi-testing.adoc[leveloffset=+1]
+
+include::modules/telco-ran-ref-config-scale-testing.adoc[leveloffset=+1]
+
+include::modules/ztp-telco-ran-software-versions.adoc[leveloffset=+1]
```

Comment:
Most of these sections are (my opinion) intended to be internal references. I would limit this section to the validated software versions. The testing results are already covered in the KPI section.

Reply:
removed everything except software